CatAnalyzer/src/LepJetsFitter.cc

```diff
@@ -10,7 +10,7 @@ namespace ttbb
   TLorentzVector tmplep, tmpnu, tmpbl, tmpbj, tmpj1, tmpj2;
   float blres, bjres, j1res, j2res, metres;
 
-  const float CSVWP = cat::WP_BTAG_CSVv2T;
+  const float CSVWP = cat::WP_DeepCSVT;
 }
 
 // full solution
```

Comment:
are you using this "fitter"? why don't you remove it?

Reply:
I'm going to remove LepJet Fitter as well as Javier's old ttbb analyzer.